project3/styles.css

```diff
@@ -0,0 +1,321 @@
+    height: 100px;
+    padding-bottom: 40px;
+    padding-top: 20px;
+    margin-left: auto;
+    margin-right: auto;
+}
+
+.clock {
+    width: 100px;
+    height: 100px;
+    padding-bottom: 40px;
+    padding-top: 20px;
+    margin-left: auto;
+    margin-right: auto;
+
+}
+
+.iconsubs {
+    padding-top: 25px;
+}
+
+.muscle {
+    width: 100px;
+    height: 100px;
+    padding-bottom: 40px;
+    padding-top: 20px;
+    margin-left: auto;
+    margin-right: auto;
+}
+
+textarea {
+    height: 120%;
+    width: 75%;
+}
+
+textarea:hover {
+    border: 1px solid #71f79f;
+}
+
+input[type="radio"],
+input[type="checkbox"] {
+    -moz-animation: none;
+    -webkit-appearance: none;
+    appearance: none;
+    border: 1px solid #cececd;
+    height: 20px;
+    width: 20px;
+}
+
+input[type="radio"] {
+    border-radius: 50%;
+}
+
+
+input[type="checkbox"]:checked {
+    background-color: #71f79f;
+    border-color: #d1d5de;
+    font-size: #cececd;
+    font-family: "Benchnine", monospace;
+    font-size: 22px;
+}
+
+input[type="radio"]:checked {
+    background-color: #71f79f;
+    border-color: #d1d5de;
+    font-size: #cececd;
+    font-family: "Benchnine", monospace;
+    font-size: 22px;
+}
+
+input[type="checkbox"]:checked+label,
+input[type="radio"]:checked+label {
+    text-decoration: underline;
+    text-decoration-color: #71f79f;
+}
+
+input[type="radio"]:hover,
+input[type="checkbox"]:hover {
+    background-color: #1e2019;
+    border: 1px solid #71f79f;
+}
+
+input[type="text"],
+input[type="email"],
+input[type="number"] {
+    font-family: "Benchnine", monospace;
+    font-size: 18px;
+    color: #1e2019;
+    display: block;
+    width: 50%;
+}
+
+input[type="text"]:focus,
+input[type="email"]:focus,
+input[type="number"]:focus {
+    color: #1e2019;
+}
+
+input[type="text"]:required:valid,
+input[type="email"]:required:valid,
+input[type="number"]:required:valid {
+    border-bottom-color: #71f79f;
+    font-family: "Benchnine", monospace;
+    font-size: 18px;
+    color: #1e2019;
+}
+
+input[type="text"]:not(:placeholder-shown):invalid,
+input[type="email"]:not(:placeholder-shown):invalid,
+input[type="number"]:not(:placeholder-shown):invalid {
+    border-bottom-color: #c61a3a;
+    font-family: "Benchnine", monospace;
+    font-size: 18px;
+    color: #1e2019;
+}
+
+p {
+    font-family: "Benchnine", monospace;
+    font-size: 18px;
+    color: #cececd;
+}
+
+h1 {
+    font-family: "Teko", sans-serif;
+    font-size: 48px;
+    color: #1e2019;
+    text-align: center;
+}
+
+.top {
+    background-color: #cececd;
+    max-width: 100%;
+    padding-bottom: 50px;
+}
+
+.caption {
+    font-family: "teko", sans-serif;
+    font-size: 26px;
+    text-align: center;
+    color: #1e2019;
+    display: block;
+    position: relative;
+    bottom: 40px;
+}
+
+h2 {
+    font-family: "Teko", sans-serif;
+    font-size: 40px;
+    color: #cececd;
+}
+
+.button {
+    padding-top: 40px;
+    padding-bottom: 50px;
+}
+
+.button-reset {
```

Comment:
Here you've combined generic styles for a utility class (`.button-reset`) with specific styles for your page. The idea behind a utility class is that it is reusable within a project and from project to project. I would keep the `.button-reset` styles as they were in [my example](https://github.com/mica-web/gd2/blob/1fa66b0aaa4ab516217ec9702adf905c198c187f/project3/styles.css#L115) and then add a new class for your specific button styles.